src/config/hintr_version

```diff
@@ -1 +1 @@
-master
+mrc-2398
```

Comment:
I am not sure why this is not master. Does that mean it's not mergeable until the hintr branch is merged ?

Reply:
Yes, that's correct.